packages/api-v2/src/hooks/useCtraderAccountsList.ts

```diff
@@ -26,8 +26,8 @@ const useCtraderAccountsList = () => {
                     ...account,
                     /** Account's currency config information */
                     currency_config: account.currency ? getConfig(account.currency) : undefined,
-                    /** The id of the cTrader account */
-                    id: account.account_id,
+                    /** The login id of the cTrader account */
+                    loginid: account.account_id,
```

Comment:
Cross-checked with components using this hook and found out that this `id` is not used anywhere and only the `login` or `account_id` endpoints are used, hence why it is changed to `loginid` to get a constant endpoint between the different CFD account types and solve type issues for missing endpoint

![image](https://github.com/binary-com/deriv-app/assets/142860499/d41840dc-8e2e-40b7-b954-e73e5fc8f080)
![image](https://github.com/binary-com/deriv-app/assets/142860499/d1677156-0ec4-4679-aece-7a095c600517)
![image](https://github.com/binary-com/deriv-app/assets/142860499/f655ed83-fbf9-4f46-ab47-2c9757a4bcdf)